products/dotnet/get-started.adoc

```diff
@@ -35,17 +35,21 @@ Finally, yes, you can install on bare metal if you wish. #{site.base_url}/produc
 
 Here’s the list of instructions to install .NET Core on Red Hat Enterprise Linux:
 
-1. Run `sudo yum install -y rh-dotnetcore10`
+1. Register your RHEL machine using your developer subscription
+2. Run `subscription-manager repos --enable=rhel-7-server-dotnet-rpms`
```

Comment:
@omajid the two ways to do a full preview that I'm aware of:
* a preview build that takes place when you open a PR or add commits to an open PR on github (see the site preview link when the CI/CD stuff runs.
* have a local copy of the development environment.   I run the site in docker containers per the README.md at the top of this repo on fedora, but it was a quite a bit of work to get going,  I wouldn't recommend going that route given how close we are to moving things to Drupal.

Spot checking your latest commit looks good.   I'd recommend just opening a PR so you get a preview build at this point.   

Let me know if you want me to give it a check/proof read.